### PR TITLE
ODK 217

### LINKED ISSUE #4926: HPO release has incorrect version IRI
The current release of HPO has version IRI `http://purl.obolibrary.org/obo/hp/releases/2019-06-17`. It should instead be something like `http://purl.obolibrary.org/obo/hp/releases/2019-06-17/hp.owl` (which doesn't resolve).

The previous release seems to also have a bad version IRI: `http://purl.obolibrary.org/obo/hp/releases/2019-06-03`. If the filename is appended to this, it does resolve: `http://purl.obolibrary.org/obo/hp/releases/2019-06-03/hp.owl`

### PR DESCRIPTION
**Pull request by matentzn**
- Updated Makefile to latest ODK
- Fixes #4926
- Fixed hp-edit.owl which had a few foreign terms (from Chebi)

@drseb check especially the diff on hp-edit.owl. I think its ok, but better have a second eye.

I deleted one EQ, which still caused an illegal equivalent class axioms (abnormal cardiovascular system = abnormal cardiovascular system morphology).

### CODE REVIEW

File: src/ontology/hp-edit.owl [hunk 1]
```diff
@@ -15043,7 +15038,6 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/RO_0000052> "inheres_in"^^xsd:string)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#shorthand> <http://purl.obolibrary.org/obo/RO_0000052> "inheres_in"^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/RO_0000052> "inheres in"^^xsd:string)
-SubObjectPropertyOf(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/RO_0002314>)
```
Comment (drseb) [inline src/ontology/hp-edit.owl]: why are we deleting this? We added this intentionally. Is this now somewhere else or is it not important anymore? @cmungall @matentzn

File: src/ontology/hp-edit.owl [hunk 2]
```diff
@@ -33752,7 +33721,6 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/HP_0001626> "HP:0001626"^^xsd:string)
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0001626> "The cardiovascular system consists of the heart, vasculature, and the lymphatic system."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0001626> "Abnormality of the cardiovascular system"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0001626> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/UBERON_0004535>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
```
Comment (drseb) [inline src/ontology/hp-edit.owl]: do we have a replacement for this? @nicolevasilevsky @matentzn

Comment (matentzn) [reply src/ontology/hp-edit.owl]: Yes, will have. Not important now; the next step in the patternisation involves fully defining all these top level classes.

File: src/ontology/hp-edit.owl (hunk 1 above)
Comment (matentzn) [reply src/ontology/hp-edit.owl]: The odk makes sure that all axioms that are not HP native are pulled in from the outside automatically. This one is being pulled from RO!

Comment (drseb) [reply src/ontology/hp-edit.owl]: Thanks for the clarification

**Pull request merged**